catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/DefaultCswRecordMap.java

```diff
@@ -215,6 +217,10 @@ public static boolean hasDefaultMetacardFieldForPrefixedString(String propertyNa
         }
     }
 
+    private static boolean isXpathPropertyName(String propertyName) {
+        return ((propertyName.indexOf('/') != -1 || propertyName.indexOf('@') != -1));
```

Comment:
I thought we should only assume it is xpath if it begins with a '/' or '@' so shouldn't we only be checking the first index?